aaelf64/aaelf64.rst

```diff
@@ -1452,35 +1452,35 @@ Thread-local storage descriptors
-  | 566        | \-         | R\_<CLS>\_TLSDESC\_ OFF\_G0\_NC | G(GTLSDESC(S+A)) - GOT           | Set a MOVK immediate field to bits [15:0] of X. No overflow check.                           |
-  +------------+------------+---------------------------------+----------------------------------+----------------------------------------------------------------------------------------------+
-  | 567        | \-         | R\_<CLS>\_TLSDESC\_ LDR         | None                             | For relaxation only. Must be used to identify an LDR instruction which loads the TLS         |
-  |            |            |                                 |                                  | descriptor function pointer for S + A if it has no other relocation.                         |
-  +------------+------------+---------------------------------+----------------------------------+----------------------------------------------------------------------------------------------+
-  | 568        | \-         | R\_<CLS>\_TLSDESC\_ ADD         | None                             | For relaxation only. Must be used to identify an ADD instruction which computes the          |
-  |            |            |                                 |                                  | address of the TLS Descriptor for S + A if it has no other relocation.                       |
-  +------------+------------+---------------------------------+----------------------------------+----------------------------------------------------------------------------------------------+
-  | 569        | 127        | R\_<CLS>\_TLSDESC\_ CALL        | None                             | For relaxation only. Must be used to identify a BLR instruction which performs an indirect   |
-  |            |            |                                 |                                  | call to the TLS descriptor function for S + A.                                               |
-  +------------+------------+---------------------------------+----------------------------------+----------------------------------------------------------------------------------------------+
+  +------------+------------+---------------------------------+----------------------------------+-------------------------------------------------------------------------------------------------------+
+  | ELF64 Code | ELF32 Code | Name                            | Operation                        | Comment                                                                                               |
+  +============+============+=================================+==================================+=======================================================================================================+
+  | 560        | 122        | R\_<CLS>\_TLSDESC\_ LD\_PREL19  | G(GTLSDESC(S+A)) - P             | Set a load-literal immediate to bits [20:2]; check -2\ :sup:`20` <= X < 2\ :sup:`20`; check X & 3 = 0.|
```

Comment:
Just a comment that you don't have to address if you disagree: as a personal preference I would have just increased of one line the height of the two lines being touched, avoiding to extend all the tables separators.

Reply:
Done. Looks better indeed.